src/components/players/player/on-air/OnAir.js

```diff
@@ -4,11 +4,11 @@ import Colors from "../../../../consts/Colors";
 import { Tiny } from "../../../text-elements/index";
 import Devices from "../../../../consts/Devices";
 
-const OnAir = () => {
+const OnAir = ({ on }) => {
```

Comment:
`isOnAir`? Might as well start the better naming practice now

Reply:
Good point! [Fixed](https://github.com/RichardPK/eh-fm/pull/44/commits/3df5982b7639ef413d244adf7598904829ba04a4)